docs/operate/get-started/supported-hardware.md

```diff
@@ -85,6 +85,26 @@ After installing `viam-server` or `viam-micro-server` on your computer or microc
 
    For details on configuring versioning and environment variables for modules, see [Modular Resource and Module Configuration Details](/operate/reference/module-configuration/).
 
```

Comment:
Maybe right before this there's a `### Configure virtual hardware components` heading where you can add that those exist? Remember this page is not about creating new modules so the sentence `You can also create a module to support new software-only functionality.` doesn't really fit. What does fit is to point out "You may come across virtual hardware components that use the same API but ..."